invenio_communities/assets/semantic-ui/less/invenio_communities/theme.less

```diff
@@ -154,3 +154,23 @@
     margin-top: 0;
   }
 }
+
+.community-description-show:hover{
+  text-decoration: none;
+}
+
+.box {
```

Comment:
sorry, naming police, shouldn't this class name be more descriptive? especially because box can be easily used elsewhere